omaha/goopdate/download_manager.cc

```diff
@@ -530,7 +533,17 @@ HRESULT DownloadManager::CachePackage(const Package* package,
   const CString package_name(package->filename());
   PackageCache::Key key(app_id, version, package_name);
 
-  HRESULT hr = package_cache()->Put(
+  HRESULT hr;
```

Comment:
Should initialize hr.

Reply:
Fixed in 2056208a294f3250d68ebb2aa2e8ca858d95ea39.